sound/soc/codecs/es8326.c

```diff
@@ -0,0 +1,854 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * es8326.c -- es8326 ALSA SoC audio driver
+ * Copyright Everest Semiconductor Co.,Ltd
+ *
+ * Authors: David Yang <yangxiaohua@everest-semi.com>
+ */
```

Comment:
Mark Brown will ask you to use C++ // comments for all the lines

Reply:
I followed the style of other codecs drivers. Is it ok?

Reply:
let's see.